frontend/server/src/DAO/QualityNominations.php

```diff
@@ -190,6 +190,68 @@ private static function getVotesForNomination(int $qualitynomination_id) {
         return $votes;
     }
 
+    /**
+     * Returns the quality seal and id of a nomination for a given problem and user.
+     *
+     * @return array{quality_seal: bool, qualitynomination_id: int}
+     */
+    public static function getReviewedData(
+        int $problemId,
+        int $userId
+    ): array {
+        $sql = "
+            SELECT
+                qn.contents,
+                qn.qualitynomination_id
+            FROM
+                QualityNominations qn
+            INNER JOIN
+                Identities i ON i.user_id = qn.user_id
+            WHERE
+                nomination = 'quality_tag' AND
+                i.identity_id = ? AND
+                qn.problem_id = ?;";
+        $params = [
+            $userId,
+            $problemId,
+        ];
+
+        $row = \Omegaup\MySQLConnection::getInstance()->GetRow($sql, $params);
```

Comment:
Acá faltaría la declaración del array obtenido de la DB:
```suggestion
        /** @var array{contents: string, qualitynomination_id: int}|null */
        $row = \OmegaUp\MySQLConnection::getInstance()->GetRow($sql, $params);
```

y antes de decodificar el json habrá que validar si no se obtuvo nada:
```php
        if (is_null($row)) {
            return [
                'quality_seal' => false,
                'qualitynomination_id' => 0,
            ];
        }
```